audit_log/middleware.py

```
@@ -129,3 +134,193 @@ def process_request(self, request):
+            # Create a mock request object for ASGI
+            from django.http import HttpRequest
+            from django.contrib.sessions.middleware import SessionMiddleware
+            from django.contrib.auth.middleware import AuthenticationMiddleware
+            
+            request = HttpRequest()
+            request.method = scope["method"]
+            request.path = scope["path"]
+            request.META = dict(scope.get("headers", []))
+            
+            # Apply session middleware
+            session_middleware = SessionMiddleware(lambda req: None)
```

Ruff (0.13.1)

Unused lambda argument: `req`

(ARG005)